medical/views/medical_patient_view.xml

```diff
@@ -1,250 +1,351 @@
-             <field name="name">medical.patient.form</field>
-             <field name="model">medical.patient</field>
-             <field name="arch" type="xml">
-              <form string="Patient">
-                    <header>
-                    </header>
-                    <sheet>
-                        <div class="oe_title" name="title">
-                            <field name="image" widget='image'
-                                class="oe_left oe_avatar" options="{'preview_image': 'image_medium', 'size': [90, 90]}"/>
-                            <h1><field name="name" placeholder="First Name Last Name" /></h1>
-                        </div>
-                        <div class="oe_right oe_button_box" name="buttons">
-                        </div>
-                        <group name="data">
-                            <group name="main">
-                                 <field name="ref"/>
-                                 <field name="identification_code"/>
-                                 <field name="medical_center_id"/>
+<odoo>
+
+    <record id="medical_patient_view_tree" model="ir.ui.view">
+        <field name="name">medical.patient.tree</field>
+        <field name="model">medical.patient</field>
+        <field name="arch" type="xml">
+            <tree string="Medical Patient">
+                <field name="name" />
+                <field name="ref" />
+                <field name="identification_code" />
+                <field name="age" />
+                <field name="gender" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="medical_patient_view_form" model="ir.ui.view">
+        <field name="name">medical.patient.form</field>
+        <field name="model">medical.patient</field>
+        <field name="arch" type="xml">
+        <form string="Patient">
+            <header />
+            <sheet>
+                <div class="oe_right oe_button_box" name="buttons" />
+                <field name="image"
+                       widget='image'
+                       class="oe_left oe_avatar"
+                       options="{'preview_image': 'image_medium', 'size': [90, 90]}" />
+                <div class="oe_title" name="title">
+                    <h1><field name="name" placeholder="First Name Last Name" /></h1>
+                </div>
+                <group name="data">
+                    <group name="main">
+                        <field name="identification_code" />
+                        <field name="gender" />
+                        <field name="is_pregnant" />
+                    </group>
+                    <group>
+                        <field name="birthdate_date" />
+                        <field name="date_death" />
+                        <field name="age" />
+                    </group>
+                </group>
+                <notebook>
+                    <page string="General Information"> 
```

Comment:
Add a name attribute to enable inheritance. In v10 I think you can only inherit without using xpath if you have a name attribute.